### PR TITLE
# Creating a Building Registration Form

### PR DESCRIPTION
**Pull request by OMOTAYO05**
## Building registration form using _html_ and _css_ 

> A building registration form website with html with various element

- `form`
- `input`
- `article`
- `legend`
- `section`

> The css is used for styling the page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,22 @@
 
 > One paragraph statement about the project.
```
Comment (Gbemisola19) [inline README.md:3]: input the value of the instruction given

File: README.md [hunk 2]
```diff
@@ -2,19 +2,22 @@
```
Comment (Gbemisola19) [inline README.md:2]: Lets add the project name

File: README.md [hunk 3]
```diff
@@ -2,19 +2,22 @@
...
 Additional description of the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
+- HTML5,
+- `form`,`label`,`input`,`fieldset`,`select`,`textarea`,`a`,
+- CSS
+- `background-color`,`width`,`height`,`border`,`font-size`,`display`,`font-family`,
```
Comment (Gbemisola19) [inline README.md:14]: make the elements a nested list

Comment (OMOTAYO05) [reply README.md:14]: done

File: README.md (hunk 2 above)
Comment (OMOTAYO05) [reply README.md:2]: done

File: README.md (hunk 1 above)
Comment (OMOTAYO05) [reply README.md:3]: Done Ma`am

**Pull request merged**